src/components/Switch/Item.tsx

```diff
@@ -2,31 +2,35 @@ import * as Switch from "@radix-ui/react-radio-group";
 import { ReactNode } from "react";
 
 import { DataAttributes } from "../types";
+import { Box, PropsWithBox } from "../Box";
 import { switchChild } from "./Switch.css";
 
-export type SwitchItemProps = {
+export type SwitchItemProps = PropsWithBox<{
   value: string;
   id: string;
   children: ReactNode;
   disabled?: boolean;
-} & DataAttributes;
+}> &
+  DataAttributes;
 
 export const SwitchItem = ({
   value,
   id,
   disabled = false,
   children,
   ...rest
-}: SwitchItemProps) => (
-  <Switch.Item
-    value={value}
-    id={id}
-    disabled={disabled}
-    className={switchChild()}
-    {...rest}
-  >
-    {children}
-  </Switch.Item>
-);
+}: SwitchItemProps) => {
+  return (
+    <Switch.Item
```

Comment:
**Suggestion:** add `asChild` prop to `Switch.Item`

Thanks to that we will have one component in HTML instead of two.

Reference: https://www.radix-ui.com/primitives/docs/components/radio-group#item